### ISSUE TITLE
When to stop?

### ISSUE THREAD

**Issue by zxpeter** [tensorpack/tensorpack#1101]
Hi Yuxin,

Recently I've applied Example/faster-rcnn(mask-rcnn) to medical image processing(breast tumor segmentation), and I have few questions:

1. I noticed that in config.py we use TRAIN.LR_SCHEDULE and TRAIN.STEPS_PER_EPOCH to control training step and epoach number；How to set suitable and appropriate parameters to fit my own dataset(have 300 samples to train and 50 to evaluate)?；
2. How to judge the model's convergence ? Do we judge that by the loss? I didn't see any related code in the project which make training stop juding by the loss. 

Thanks a lot!

**Comment by ppwwyyxx** [comment]
These are machine learning questions are not related to tensorpack. As mentioned in the issue template we do not answer such questions.

**Comment by zxpeter** [comment]
Or could you please just simply tell me in tensorpack/example/fasterrcnn, do we use loss control instead of only use TRAIN.LR_SCHEDULE and TRAIN.STEPS_PER_EPOCH to control training step?

**Comment by ppwwyyxx** [comment]
It does not use loss control
